src/backend/distributed/sql/citus--11.0-4--11.1-1.sql

```diff
@@ -79,3 +79,59 @@ DROP FUNCTION pg_catalog.get_all_active_transactions(OUT datid oid, OUT process_
 
 DROP FUNCTION pg_catalog.isolate_tenant_to_new_shard(table_name regclass, tenant_id "any", cascade_option text);
 #include "udfs/isolate_tenant_to_new_shard/11.1-1.sql"
+
+CREATE TYPE citus.citus_job_status AS ENUM ('scheduled', 'running', 'finished', 'cancelled', 'failing', 'failed');
+ALTER TYPE citus.citus_job_status SET SCHEMA pg_catalog;
+
+CREATE TABLE citus.pg_dist_background_jobs (
```

Comment:
All our table names use the singular naming format, let's stay consistent:
```suggestion
CREATE TABLE citus.pg_dist_background_job (
```

same for pg_dist_background_tasks